lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -13,6 +13,10 @@ function canMultiplyLikeTermConstantNodes(node) {
     return false;
   }
 
+  if (args.every(arg => ConstantOrPowerTerm.getExponentNode(arg).value === 1)) {
```

Comment:
mostly cosmetic, but if you had `6^1 * 6^1` it wouldn't add the exponents, it'd just ignore them. I was going to comment on that, but I think that'd never actually happen because of the `_^1 => _` rule that gets applied first